sys/mips/include/param.h

```diff
@@ -225,8 +225,8 @@
 /*
  * Mach derived conversion macros
  */
-#define	round_2mpage(x)		(((x) + PDRMASK) & ~PDRMASK)
-#define	trunc_2mpage(x)		((x) & ~PDRMASK)
+#define	round_2mpage(x)		__builtin_align_down(((x) + PDRMASK), PDRSIZE)
```

Comment:
Isn't this just `__builtin_align_up(x, PDRSIZE)`?

Reply:
Indeed